policy/modules/system/systemd.te

```diff
@@ -337,6 +337,8 @@ fs_list_efivars(systemd_efi_generator_t)
 # fstab generator local policy
 #
 
+allow systemd_fstab_generator_t self:capability { dac_override dac_read_search };
```

Comment:
Please drop these and below generator changes until the generator domains are merged.

Reply:
Done.